projects/packages/plugin-deactivation/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "jetpack-js-tools/tsconfig.base.json",
+	"include": [ "src/assets/js/**/*" ],
+	"exclude": [ "node_modules/*", "src/assets/dist/*" ],
```

Comment:
I think the "exclude" is unnecessary, you would only need if there were files in the "include" paths that should be excluded. Since neither of these are, there should be no need to mention them.

Reply:
Done in 7fd6e02401.